PowerPointLabs/PowerPointLabs/ActionFramework/Action/CropLabActionHandler.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Linq;
+using System.Text.RegularExpressions;
+
+using Microsoft.Office.Interop.PowerPoint;
+
+using PowerPointLabs.ActionFramework.Common.Interface;
+using PowerPointLabs.CropLab;
+
+using Office = Microsoft.Office.Core;
+
+namespace PowerPointLabs.ActionFramework.Action
+{
+    abstract class CropLabActionHandler : ActionHandler
+    {
+
+        protected bool VerifyIsSelectionValid(Selection selection)
```

Comment:
you may consider moving some of these generic methods (e.g. those you feel that can be re-used by other features, not just Crop Lab) to a utility class, so that the functions can be shared with other features